devito/compiler.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, openmp=False):
         self.openmp = openmp
         self.pragma_ivdep = [Pragma('ivdep')]
         self.pragma_nontemporal = []
-        self.pragma_aligned = []
+        self.pragma_aligned = "omp simd aligned(%s:%s)"
```

Comment:
The use of `%s` without replacement value looks wrong. Can we move the `(%s:%s)` part of the string to the caller?